src/chess/bitboard.h

```diff
@@ -240,10 +264,15 @@ class Move {
   }
 
  private:
-  BoardSquare from_;
-  BoardSquare to_;
-  Promotion promotion_ = Promotion::None;
-  bool castling_ = false;
+  uint16_t data_ = 0;  // First bit castling, next 3 promotion, next 6
+  // fromsquare, next 6 tosquare.
+
+  enum MASKS_ : uint16_t {
+    TO_MASK_ = 0b0000000000111111,
```

Comment:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Enum names should be written as constants: `kToMask`. Like-macro `TO_MASK` is also acceptable, but without trailing underscore.

Reply:
That's my bad, thanks for the correction

Reply:
Done